src/FhirLoader.Common/FhirResourceClient.cs

```diff
@@ -44,20 +44,22 @@ public BundleClient(string baseUrl, ILogger logger, string? tenantId = null, Asy
             _resiliencyStrategy = resiliencyStrategy ?? DefaultResiliencyStrategy();
         }
 
-        public async Task Send(ProcessedBundle bundle, Action<int, long>? metricsCallback = null, CancellationToken? cancel = null)
+        public async Task Send(ProcessedResource bundle, Action<int, long>? metricsCallback = null, CancellationToken? cancel = null)
         {
-            var content = new StringContent(bundle.BundleText!, Encoding.UTF8, "application/json");
+            var content = new StringContent(bundle.ResourceText!, Encoding.UTF8, "application/json");
             HttpResponseMessage response;
 
+            var requestUri = !string.IsNullOrEmpty(bundle.ResourceType) ? $"/{bundle.ResourceType}" : string.Empty;
+
             var timer = new Stopwatch();
             timer.Start();
 
             try
             {
-                _logger.LogTrace($"Sending {bundle.BundleCount} resources to {_client.BaseAddress}...");
+                _logger.LogTrace($"Sending {bundle.ResourceCount} resources to {_client.BaseAddress}...");
```

Comment:
Can we add `bundle.ResourceType` to the message so users know what is being sent?

Reply:
Do you mean to print ResourceType as "bundle" for ndjson and bundle and for profile will read from processedResource.ResourceType?

Reply:
Yes! Exactly. So users know what they have sent. Super useful for profiles.

Reply:
Added Resource Type to the message string.